README.md

```diff
@@ -121,3 +121,46 @@ You can also pass nativefier flags, and mount additional volumes to provide loca
 ## License
 
 [MIT](LICENSE.md)
+
+## TS: Minimum to release beta
```

Comment:
> When merging to master and releasing, it seems like everything _should_ work on Travis CI when master is tagged, but let me know so I can keep an eye on it should something go wrong :P

@jiahaog okay.

One more related thing maybe for later: it would be cool to be able to publish pre-releases to an npm `beta` channel. Users could test-drive this build with `npm i nativefier@beta` rather than having to do a dev build.